NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf

```diff
@@ -269,13 +269,21 @@
   % \fi
 }
 
+% Empty placeholder for cover position 1-2
+\ntaddtocover[]{1-2}{%
+}
+
+% Empty placeholder for cover position N-1
+\ntaddtocover[]{N-1}{%
+}
+
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% BACK COVER
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % Print sponsor logos
-\ntaddtocover[vspace=1, halign=l]{N-2}{%
+\ntaddtocover[vspace=1cm, halign=l]{N-2}{%
   \foreach \myi in {1, ..., 9} {%
```

Comment:
As above, this suggestion is wrong. That is not supposed to be `1cm` but `1` as it is. The number without a unit means `\vfill`, which is great for relative vertical spacing. If you replace it with `1cm` it will be out of format.